codegen/src/main/resources/SpringCloud/consumer/openfeign/applicationYml.mustache

```diff
@@ -0,0 +1,11 @@
+spring:
+  application:
+    name: {{serviceId}}
```

Comment:
same as ServiceComb mustache,both of provider and consumer use the "serviceId" as value, it will generate the same service name while we
use ServiceType in all

Reply:
This problem does exist, thank you for your inspection. I will fix it in the next commit later.